tests/clar.c

```diff
@@ -389,9 +396,6 @@ clar_parse_args(int argc, char **argv)
 		case 'v':
 			_clar.report_suite_names = 1;
 			break;
-
-		default:
-			clar_usage(argv[0]);
```

Comment:
Can you keep a default here?  An `assert` is fine, just something so that in case somebody adds an option to the `strchr` validation (above) during development that we won't simply barge on through without validation.